features/importing/work_import_da.feature

```diff
@@ -11,50 +11,35 @@ Feature: Import Works from deviantart
       And I fill in "urls" with "http://bingeling.deviantart.com/art/Flooded-45971613"
     When I press "Import"
     Then I should see "Preview"
-      And I should see the image "src" text "Flooded_by_bingeling.jpg"
+      And I should see the image "src" text "http://orig03.deviantart.net/4707/f/2007/004/a/7/flooded_by_bingeling.jpg"
       And I should see "Digital Art" within "dd.freeform"
       And I should see "People" within "dd.freeform"
       And I should see "People" within "dd.freeform"
       And I should see "Vector" within "dd.freeform"
       And I should see "Published:2007-01-04"
-      And I should see "Flooded" within "h2.title"
+      # Importer picks up artist name as title instead of actual title
```

Comment:
Not that I'm aware of - would you like me to check and open one if not? Is it something that we're likely to fix anytime soon?